2018/05/21 12:10:37, Exception was thrown: Registraition of the following item failed: 2222
se.kth.iv1201.pos.controller.OperationFailedException: Registraition of the following item failed: 2222
	at se.kth.iv1201.pos.controller.Controller.enterItemId(Controller.java:72)
	at se.kth.iv1201.pos.view.View.sampleExecution(View.java:75)
	at se.kth.iv1201.pos.startup.Main.main(Main.java:40)
Caused by: se.kth.iv1201.pos.dbhandler.DatabaseErrorException: Database exception:  2222
	at se.kth.iv1201.pos.dbhandler.InventorySystem.matchItemId(InventorySystem.java:71)
	at se.kth.iv1201.pos.dbhandler.InventorySystem.getItem(InventorySystem.java:41)
	at se.kth.iv1201.pos.dbhandler.ExternalSystem.getItem(ExternalSystem.java:39)
	at se.kth.iv1201.pos.controller.Controller.enterItemId(Controller.java:63)
	... 2 more
2018/05/21 12:10:37, Exception was thrown: Registraition of the following item failed: 2222
se.kth.iv1201.pos.controller.OperationFailedException: Registraition of the following item failed: 2222
	at se.kth.iv1201.pos.controller.Controller.enterItemId(Controller.java:72)
	at se.kth.iv1201.pos.view.View.sampleExecution(View.java:75)
	at se.kth.iv1201.pos.startup.Main.main(Main.java:41)
Caused by: se.kth.iv1201.pos.dbhandler.DatabaseErrorException: Database exception:  2222
	at se.kth.iv1201.pos.dbhandler.InventorySystem.matchItemId(InventorySystem.java:71)
	at se.kth.iv1201.pos.dbhandler.InventorySystem.getItem(InventorySystem.java:41)
	at se.kth.iv1201.pos.dbhandler.ExternalSystem.getItem(ExternalSystem.java:39)
	at se.kth.iv1201.pos.controller.Controller.enterItemId(Controller.java:63)
	... 2 more
